.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Make SPA Responsive for Desktop, Laptop, Mobile

Reply:
Resolved